### PR TITLE
ResBundle class copied from ilib v14.15.0

### PR DESCRIPTION
**Pull request by ehoogerbeets**
- converted to ESM classes
- now can load resources from the same global roots that ilib data is loaded

### CODE REVIEW

**Comment by ehoogerbeets** [comment]
@gouniLee This one is ready for review now. Please have a look.

File: locale/eu/ES/pseudomap.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (gouniLee) [inline locale/eu/ES/pseudomap.json]: Should we treat eu-ES and ps-AF locales are pseudo locales?
To align loctool, I think these locales shouldn't be pseudo locales. no?

Comment (ehoogerbeets) [reply locale/eu/ES/pseudomap.json]: updated, please re-review

**Review by gouniLee — APPROVED** [review]

**Comment by gouniLee** [comment]
oh forgot to say it. It's just my thought. 
Without seeing  package.json, I just did `npm run build` the `npm run test`
and I faced an error due to the locales.js file missing. 
After checking the packages.json files now I could execute them correctly in order.  
I'm not sure if it would be better if we could provide suggestions for executing orders or if it's just too much information.

**Pull request merged**